fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/Key.java

```diff
@@ -66,9 +66,13 @@
         private Expressions() {}
 
         /**
-         * Create an expression of a single scalar (i.e. not repeated) field.
+         * Create an expression of a single scalar (i.e. not repeated) field. This sets the {@link Evaluated.NullStandin}
+         * to {@link Evaluated.NullStandin#NULL}. See {@link #field(String, KeyExpression.FanType, com.apple.foundationdb.record.metadata.Key.Evaluated.NullStandin)}
```

Comment:
This is using the fully qualified package name here (and elsewhere) because of what I think is a Javadoc bug.